js/api.ts

```diff
@@ -252,13 +252,12 @@ export function updateReportSettingsInPortal(data: any) {
       body: JSON.stringify(data),
     }).then(checkStatus);
   } else {
-    // tslint:disable-next-line:no-console
     console.warn("No OFFERING/REPORT_URL. Faking put method.");
     return new Promise(resolve => resolve({}));
   }
 }
 
-export function feedbackSettingsFirestorePath(sourceKey: string, instanceParams?: { platformId?: string, resourceLinkId?: string }) {
+export function feedbackSettingsFirestorePath(sourceKey: string, instanceParams?: { platformId?: string; resourceLinkId?: string }) {
```

Comment:
Is the `;` right here?

Reply:
It's an interface definition, not a JavaScript object so yes semicolon is correct, although it was not immediately obvious to me, either. It was found by the linter.